packages/ozone/src/team/index.ts

```diff
@@ -197,11 +198,13 @@ export class TeamService {
       isAdmin || member?.role === 'tools.ozone.team.defs#roleModerator'
     const isTriage =
       isModerator || member?.role === 'tools.ozone.team.defs#roleTriage'
+    const isVerifier = member?.role === 'tools.ozone.team.defs#roleVerifier'
```

Comment:
Should admins also have this flag?  That would be consistent with `adminToken()` in `auth-verifier`.